studio/components/OpenSharedLink.tsx

```diff
@@ -0,0 +1,9 @@
+import Link from "next/link";
+
+export const OpenSharedLink = () => {
+  return (
+    <Link href="http://localhost:3000/shared">
```

Comment:
This link only works locally, right? It should be relative instead
```suggestion
    <Link href="/shared">
```